skabase/SKALogger/test/SKALogger_test.py

```diff
@@ -8,23 +8,24 @@
 #########################################################################################
 """Contain the tests for the SKALogger."""
 
-# Path
+
+# standard imports
 import sys
 import os
 
-path = os.path.join(os.path.dirname(__file__), os.pardir)
-sys.path.insert(0, os.path.abspath(path))
-
 # Imports
```

Comment:
The formatting of this section is not consistent with the other changes in this PR.

Reply:
It looks consistent all across now